fmriprep/interfaces/freesurfer.py

```diff
@@ -190,7 +192,42 @@ def _run_interface(self, runtime):
         return runtime
 
 
-class PatchedConcatenateLTA(ConcatenateLTA):
+class TruncateLTA(object):
+    # Use a tuple in case some object produces multiple transforms
+    lta_outputs = ('out_lta_file',)
+
+    def _post_run_hook(self, runtime):
+
+        outputs = self._list_outputs()
```

Comment:
This should be

```Python
outputs = self.aggregate_outputs(runtime)
```

`_list_outputs()` is the predicted outputs and may be called at any time, while `aggregate_outputs()` is the actual outputs, and must be called after `_run_interface()`.